dmfsi.dmr

```diff
@@ -0,0 +1,30 @@
+# DMOD Resource File for dmfsi
```

Comment:
Przecież ty najważniejszego nie zrobiłeś czyli nie zaktualizowałeś flow release CI żeby to w ogóle użyć.